test/Microsoft.DotNet.Docker.Tests/ImageDescriptor.cs

```diff
@@ -2,16 +2,20 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
+
 namespace Microsoft.DotNet.Docker.Tests
 {
     public class ImageDescriptor
     {
         private string runtimeDepsVersion;
         private string sdkVersion;
+        private string sdkOsVariant;
 
         public string Architecture { get; set; } = "amd64";
         public string DotNetCoreVersion { get; set; }
         public string OsVariant { get; set; }
+        public bool IsArm { get => String.Equals("arm", Architecture, StringComparison.OrdinalIgnoreCase); }
```

Comment:
Consider moving up in alphabetical order.

Reply:
Ok, and how about `sdkVersion` and `sdkVersion`?

Reply:
For now, just moving `IsArm` between `DotNetCoreVersion` and `OsVariant`.